topics/jwt.md

```diff
@@ -14,6 +16,10 @@
 <include from="lib.topic" element-id="native_server_not_supported"/>
 </tldr>
 
+<link-summary>
+The %plugin_name% plugin allows you to perform validations of JSON Web Tokens for authentication purposes. 
```

Comment:
Isn't the purpose to authenticate the clients using JWT?

Reply:
You're right. I phrased it like that to be more specific but "authenticate clients using JWT" sounds better!